### DISCUSSION TITLE
Stepper Component

### DISCUSSION THREAD

**Discussion by ishah-twilio** [twilio-labs/paste#623] [General] [answered]
Do we have a stepper component (in sketch - library) for wizard?

**Comment by kate-porter** [comment, marked as answer]
Hey @ishah-twilio! We do not have a stepper component yet, and we likely won't have one particularly soon. But you can compose your own using [Box and Text primitives](https://paste.twilio.design/tokens/how-to-compose-custom-ui-with-tokens/). Box and Text allow you to compose custom components that still use Paste.

I know you've already seen this video, but I want to share it here so other folks can find it if they have a similar question in the future - this video covers how to compose custom compositions in Sketch: https://www.loom.com/share/7f3fe097ad9d4c5abf2a5cd0cdd0846a. Conveniently enough, the example used in the video is a custom stepper component. ;) This is just an example, so your stepper doesn't need to look anything like what's built in the video if your use case warrants a different approach.

Please reach out if you need help putting the custom composition together in Sketch. This is something folks haven't done a ton yet, so we are here to help and also very open to feedback about how things could be easier.

Simon also put together a code sandbox for the same custom composition that we build in that video. Your engineers can reference this when they get to the build stage: https://codesandbox.io/s/progress-path-5io3t?file=/src/ProgressPath.tsx.
